### PR TITLE
Fix minor typos and grammatical errors in README.md

### PR DESCRIPTION
**Pull request by virejdasani**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,9 +22,9 @@
 
 </div>
 
-**LogicRs** is an easy-to-use, free and open-source simulator for logical circuits/diagrams written in rust.
+**LogicRs** is an easy-to-use, free, and open-source simulator for logical circuits/diagrams written in Rust.
```
Comment (Spydr06) [inline README.md:25]: I'm not sure weather the comma actually belongs there before "and"

File: README.md [hunk 2]
```diff
@@ -35,8 +35,8 @@ LogicRs can be downloaded as a prebuilt package from the [releases](https://gith
 - `.AppImage` package (Linux)
 - `.zip` portable installation (Windows)
 
-Alternatively, you can download this repository and build LogicRs from source. 
-Please refer to [INSTALL.md](./INSTALL.md) for build instructions and information about compatability
+Alternatively, you can download this repository and build LogicRs from the source.
```
Comment (Spydr06) [inline README.md:38]: > build LogicRs from **the** source
Here, "source" refers to the source code, where it's common to say either "from source" or "from the source code".

**Review by Spydr06 — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (virejdasani) [reply README.md]: I see, I have made the appropriate changes :)

Comment (Spydr06) [reply README.md]: Good, ty

**Comment by Spydr06** [comment]
Thank you

**Review by Spydr06 — APPROVED** [review]

**Pull request merged**